tox_pip_extensions.py

```diff
@@ -41,14 +41,24 @@ def _assert_true_value(k, v):
     else:
         raise ValueError('Expected "true" but got {} = {!r}'.format(k, v))
 
+def _get_tox_section(cfg):
+    exc_info = (None, None, None)
+    for section in ["tox:tox", "tox"]:
+        try:
+            return cfg.items(section)
+        except six.moves.configparser.NoSectionError:
+            exc_info = sys.exc_info()
+    six.reraise(*exc_info)
+
+
 
 @hookimpl
 def tox_configure(config):
     cfg = six.moves.configparser.ConfigParser()
     cfg.read(str(config.toxinipath))
     configured = tuple(sorted(
```

Comment:
I'm not sure why this re-parses the config, it looks like the config is available here:

```python
sections = config._cfg.sections
config = (sections.get("tox:tox", {}) | sections.get("tox", {}))
tox_pip_extensions_ext_venv_update = config["tox_pip_extensions_ext_venv_update"]
tox_pip_extensions_ext_pip_custom_platform = config["tox_pip_extensions_ext_pip_custom_platform "]
```

Reply:
"I'm not sure why, it's available in a private attribute" -- well hmm :)